### ISSUE TITLE
Read Verilog pinlist

### ISSUE THREAD

**Issue by drom** [ameetgohil/signalflip-js#9]
Do you plan to generate a wrapper for the arbitrary verilog module?
I have started this pinlist reader library https://github.com/drom/verilog-pinlist
It uses this Verilog parser https://github.com/tree-sitter/tree-sitter-verilog
Will it be useful?

**Comment by ameetgohil** [comment]
yes, absolutely. right now, i'm using ben's script to get all the pins and generate the wrapper with templates

**Comment by ameetgohil** [comment]
@drom Is the pinslist reader library ready to use?

**Comment by ameetgohil** [comment]
Verilator generates a header file called V<dut>.h which has signal width, direction, name which are easily extracted using regex. Here's the regex i used: ``` /VL_([^\(]*)\(([^\,\)]*),([^\D]*),([^\D]*)/g```
This way i don't have to worry about parameters or defines b/c verilator figures all that out.  
See https://github.com/ameetgohil/signalflip-js/blob/master/src/GenerateWrapper.js

**Comment by drom** [comment]
Could you create node bindings, that would wrap any verialted module and then discover pinlist dynamically?

**Comment by ameetgohil** [comment]
Yes, that's definitely feasible. I'll get on that this week
